kubejs/server_scripts/base/featrues/dye.js

```diff
@@ -88,23 +88,61 @@ if (feature('dye mixing')) {
     //red_dye + honey = orange dye
     addFilling('orange_dye', 'red_dye', '50x create:honey')
     //green + milk = lime
-    addFilling('lime_dye', 'green_dye', '50x minecraft:milk')
+    addFilling('lime_dye', 'green_dye', '10x minecraft:milk')
     //blue + milk = light blue
-    addFilling('light_blue_dye', 'blue_dye', '50x minecraft:milk')
+    addFilling('light_blue_dye', 'blue_dye', '10x minecraft:milk')
     //black + milk = gray
-    addFilling('gray_dye', 'black_dye', '50x minecraft:milk')
+    addFilling('gray_dye', 'black_dye', '10x minecraft:milk')
     //gray + milk = light gray
-    addFilling('light_gray_dye', 'gray_dye', '50x minecraft:milk')
+    addFilling('light_gray_dye', 'gray_dye', '10x minecraft:milk')
     //green + water = cyan
     addFilling('cyan_dye', '2x green_dye', 'water')
     //red + milk = pink
-    addFilling('pink_dye', 'red_dye', '50x minecraft:milk')
+    addFilling('pink_dye', 'red_dye', '10x minecraft:milk')
     //red + water = purple
     addFilling('purple_dye', '2x red_dye', 'water')
     //white + water = light blue
     addFilling('light_blue_dye', '2x white_dye', 'water')
```

Comment:
With the `addFilling` would it be best to use back ticks and insert numeric values in so they can easily be modified in the future so it will be easier to rebalance if needed e.g.
```js
addFilling('light_blue_dye', `${water_dye_amount}x white_dye`, 'water')
//
addFilling('pink_dye', 'red_dye', `${milk_dye_amount}x minecraft:milk`)
```

Reply:
hm.. I feel like if we go that route there are a lot of other things to change, I don't think I agree. For things that are not cross feature we don't need to do that